.insomnia/Request/req_b0483d6f0ea34b7eba74698e529ae1d1.yml

```diff
@@ -0,0 +1,147 @@
+_id: req_b0483d6f0ea34b7eba74698e529ae1d1
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1676745409437
+created: 1676743495518
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/copy"
+name: Copy
+description: >-
+  Copy articles from one review to another based on specified filters. 
+
+
+  Indiciate the target review to copy by specifying it in the request body:
+
+
+  - `target_review`: (review id to copy to)
+
+
+  Filters are passed in through the request body and are described below:
+
+
+  - `start`: Number of initial articles to skip when retrieving. Note that this depends on the sort order specified by the `order` parameter.
+
+  - `length`: The number of articles to retrieve.
+
+  - `order[0][column]`: The column to sort the articles by. If not specified, defaults to article id. Values are as follows:
+      - 0 = article id
+      - 1 = date
+      - 2 = title
+      - 3 = authors
+      - 4 = rating
+  - `order[0][dir]`: The direction to sort, either `asc` or `desc`.
+
+  - `search[value]`: Specify a keyword for retrieving articles using full-text search.
+
+  - `extra`: This is a dictionary that contains facet-based filter parameters, which are described below. Parameters ending with `[]` can contain multiple values.
```

Comment:
If this is a copy from the /results endpoint, I suggest referring to it instead of pasting a duplicate description here and having to maintain multiple copies of the same instructions.